accelerator/core/pow.c

```diff
@@ -101,9 +102,11 @@ status_t ta_pow(const bundle_transactions_t* bundle, const flex_trit_t* const tr
     ctrunk = iota_flex_digest(tx_trits, NUM_TRITS_SERIALIZED_TRANSACTION);
     tx = (iota_transaction_t*)utarray_prev(bundle, tx);
     free(nonce);
+    nonce = NULL;
   } while (cur_idx != 0 && tx != NULL);
 
 done:
   free(ctrunk);
+  free(nonce);
```

Comment:
`nonce` should already be `NULL` here.

Reply:
When occurring error, the program would jump to lanel `done`. And in this scenario, the jumping would cause memory leak

Reply:
The error occurs when `nonce` equal to `NULL`, but it is fine here since it won't free the some memory location twice.